### ISSUE TITLE
Export spatial structure containers with individual and expressions

### ISSUE THREAD

**Issue by rwmcintosh** [Open-Systems-Pharmacology/MoBi#1145]
There needs to be a workflow to export some combination of expression profiles, individual parameters, and container structure.

**Comment by Yuri05** [comment]
export both (savin expr profiles missing)

**Comment by rwmcintosh** [comment]
When user selects save or export, open a dialog and allow them to select an individual and expression profile. They can select neither to just export the container.

**Comment by PavelBal** [comment]
Additional: Multiple expression profiles can be selected.

**Comment by rwmcintosh** [comment]
We will have to change how the export/import works when a user includes individual and expression.

When we added an individual, we created this new container as a clone of the original, adding parameters to the cloned container container, and then exporting.

In this case, to keep that pattern, we would have to add some new containers for the molecules and then fill those and I'm not really sure that the new container would be usable in a simulation with those containers already added.

I'm going to change it so that when you select an individual or expressions, you get the container exported as well as a parameter values building block. When you import it, you'll have to confirm or decline to add the parameter values to the building block of the same module. If the module does not have a PV building block a new one will be created.

@PavelBal @Yuri05 @msevestre ?
